scheduler/scheduler-smartproxy/src/test/java/functionaltests/TestSmartProxy.java

```diff
@@ -311,7 +308,6 @@ protected void submitJobWithDataAndWaitToFinish(String localInputFolderPath, Str
         eventListener.setJobID(id);
 
         Thread.sleep(1000);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this use of "Thread.sleep()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2925)